doc/source/whatsnew/v0.21.0.txt

```diff
@@ -38,6 +38,7 @@ Other Enhancements
 - :func:`read_feather` has gained the ``nthreads`` parameter for multi-threaded operations (:issue:`16359`)
 - :func:`DataFrame.clip()` and :func: `Series.cip()` have gained an inplace argument. (:issue: `15388`)
 - :func:`crosstab` has gained a ``margins_name`` parameter to define the name of the row / column that will contain the totals when margins=True. (:issue:`15972`)
+- ``pd.read_excel()`` has a ``nrows`` parameter (:issue:`16645`)
```

Comment:
you can make another entry (with this PR number), in api_breaking section that the kwargs are re-aranged to match pd.read_csv

use ``:func:`read_excel` ``